src/runtime_src/core/edge/drm/zocl/zocl_xclbin.c

```diff
@@ -1021,7 +1021,11 @@ zocl_xclbin_hold(struct drm_zocl_dev *zdev, const xuid_t *id)
 {
 	xuid_t *xclbin_id = (xuid_t *)zocl_xclbin_get_uuid(zdev);
 
-	WARN_ON(uuid_is_null(id));
+	// check whether uuid is null or not
```

Comment:
code is clear. No need to comment at all.

Reply:
Thanks Min. Will remove the comment in next PR.